featurebyte/models/feature_table_cache.py

```diff
@@ -0,0 +1,55 @@
+"""
+This module contains Feature Table Cache related models
+"""
+from typing import List
+
+import pymongo
+
+from featurebyte.models.base import (
+    FeatureByteCatalogBaseDocumentModel,
+    PydanticObjectId,
+    UniqueConstraintResolutionSignature,
+    UniqueValuesConstraint,
+)
+
+
+class FeatureTableCacheModel(FeatureByteCatalogBaseDocumentModel):
+    """
+    Model for feature table cache
+
+    id: PydanticObjectId
+        FeatureTableCache id of the object
+    observation_table_id: PydanticObjectId
+        Observation Table Id
+    name: str
+        Name of the Feature Table Cache. Also used as name of the table in DWH
+    features: List[str]
+        List of feature names which are stored in this feature table cache
+    """
+
+    observation_table_id: PydanticObjectId
+    name: str
```

Comment:
If this field refers to the table name, I think would be good to rename it to something like `table_name` explicitly.